src/polyfills.ts

```diff
@@ -0,0 +1,15 @@
+// IE9, IE10 and IE11 requires all of the following polyfills.
+// import 'core-js/es6/symbol';
+import 'core-js/es6/object';
```

Comment:
These are needed?

Reply:
Nope, that was leftover from the template. Removed.